src/ProgressCircle/index.js

```diff
@@ -17,18 +17,17 @@ const showBusyContent = (busy) => {
   return false
 }
 
-const ProgressCircle = ({ variant, value, busy, label, className, ...props }) => {
+const ProgressCircle = ({ variant, value, description, busy, label, className, ...props }) => {
 
   const circleStyle = {
     strokeDashoffset: progressCircleValue(value)
   }
 
-
-
   return (
     <div className={cx('ola_progressCircle', variant && `is-${variant}`, {'is-busy': busy}, className)} {...props}>
-      { !busy && label &&  <span className="ola_progressCircle-label">{label}</span> }
+      { !busy && label && <span className="ola_progressCircle-label">{label}</span> }
       { !busy && (value !== null) &&  <strong className="ola_progressCircle-value">{value}<span>%</span></strong> }
+      { !busy && value !== null && description !== null && (typeof description === 'string' ? <span className="ola_progressCircle-label">{description}</span> : description) }
```

Comment:
Descriptions should be always in a `.ola_progressCircle-description` container, so we can place it correctly using CSS, independently of whether is a string or any other element. Something like:

```suggestion
      { !busy && description && <div className="ola_progressCircle-description">{description}</div> }
```

Reply:
@oscarotero ok, that's good, but if we want to put as description a Ola Tag component, the check if the property is a string would have to be kept

Reply:
Why we have to make this check? `{description}` can be a string or a node. See how this was done in Field: https://github.com/marketgoo/Ola/blob/development/src/Field/index.js#L41

Reply:
Yes, but if you dont check it, you are put a node inside of a span tag, with a wrapper class, I dont think that is correct.

Reply:
Ok, understand. This is why I replaced the `span` for a `div`, in order to allow any kind of elements

Reply:
@oscarotero ok, I saw. I pushed that changes. Please, check it and if you look it ok, merge and could we deploy a new version?